cmd/resource-topology-exporter/main.go

```diff
@@ -34,6 +36,13 @@ func main() {
 		log.Fatalf("failed to get podresources client: %v", err)
 	}
 
+	err = prometheus.InitPrometheus()
+	if err != nil {
+		log.Fatalf("failed to deploy prometheus server: %v", err)
+	}else {
+		log.Printf("prometheus server deployed successfully")
```

Comment:
this should be a debug log; additionally, I's use "started" rather than "deployed"

Reply:
The standard log library doesn't have a debug log by default. What other alternative do we have?

Reply:
just remove the line then